tests/foreman/ui/test_activationkey.py

```diff
@@ -320,7 +320,7 @@ def test_negative_create_with_invalid_limit(self):
                         limit=limit,
                     )
                     self.assertIsNotNone(self.activationkey.wait_until_element(
-                        locators['ak.invalid_limit']))
+                        locators['hostcollection.invalid_limit']))
```

Comment:
Nope I am not recommending to duplicate locator.  May be you can put them under `common_locators` so we dont need to refer an other entity say entity B while specifically testing entity A.